atlp-rwanda/script-heroes-bn-backend
# #177284997 Sort chat messages in descending order

#### What does this PR do?
Order Chat messages in descending order
#### Description of Task to be completed?
- Add one this line in `controllers/chat.js` file `order: [['id', 'DESC']]`
#### How should this be manually tested?
- Clone the repo git clone `https://github.com/atlp-rwanda/script-heroes-bn-backend.git`
- Switch your working directory `cd script-heroes-bn-backend`
- Checkout the branch git checkout `fix-sort-chat-messages-by-desc`
- Run tests `npm run test`
- On Postman, run a GET request on http://localhost:3000/api/chat
- Check if messages stat from the last posted

#### What are the relevant pivotal tracker stories?
[#177284997](https://www.pivotaltracker.com/story/show/177284997)
